dev-support/docker/Dockerfile_aarch64

```diff
@@ -17,7 +17,7 @@
 # Dockerfile for installing the necessary dependencies for building Hadoop.
 # See BUILDING.txt.
 
-FROM ubuntu:bionic
+FROM ubuntu:focal
```

Comment:
Upgrading from bionic to focal in a point release sounds aggressive. Dockerfile (for x86_64) still uses bionic in branch-3.3.